front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -144,12 +145,12 @@ const Map = ({
 
   const onFeatureClick = (e: MapLayerMouseEvent) => {
     if (preventPointSelection) return;
-    const result = getMapMouseEventNearestFeature(e, { layersId: ['chartis/tracks-geo/main'] });
+    const result = getMapMouseEventNearestFeature(e);
```

Comment:
As you have removed the layers which are clickable, I can select all lines and points (signals, dectectors etc)

Reply:
Actually, I think you can keep the layersId you can interact with and just add the operationalPoints layers like this:
```suggestion
    const result = getMapMouseEventNearestFeature(e, {
      layersId: [
        'chartis/tracks-geo/main',
        'chartis/osrd_operational_point/geo',
        'chartis/osrd_operational_point_yardname/geo',
        'chartis/osrd_operational_point_name_short/geo',
        'chartis/osrd_operational_point_name/geo',
        'chartis/osrd_operational_point_kp/geo',
      ],
    });
```
If you do so, the user will be able to interact with an op even if it is covered by a signal, or you will be able to interact with the track even if it is covered by electrification or speed limit